### DISCUSSION TITLE
after a while installing security onion alerts stop working

### DISCUSSION THREAD

**Discussion by pavlos1982** [Security-Onion-Solutions/securityonion#5375] [General] [answered]
I have tried the installation both a vm and a separate machine, I still get the error did not complete please see the logs, however security onion run successfully and alerts works for a while then it stop working all together see screenshot, please help, thanks

I also noticed when this alert comes up: Listened ports status (netstat) changed (new port opened or closed) it seems to stop showing alerts after.

just to add when logged in Kibana the alerts show except on security onion not showing any alert

**Comment by dougburks** [comment]
I don't see any screenshot. Please review https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and include the following information:

- Security Onion version as seen in the lower left corner of SOC and in /etc/soversion. For example: 2.3.70
- Install source. Did you install from our Security Onion ISO image or did you perform a network installation?
- If network install, did you install from CentOS 7 or Ubuntu 18.04?
- Install type. ex. eval, standalone, forward node, search node, etc
- Does so-status show all services running?
- Do you get any failures when you run salt-call state.highstate from the node?
- Does the grid status show any failures?
- Explain your issue. For example: Installation fails when I select this series of options...
- Provide applicable logs. For example: if you are having problems during soup, provide /root/soup.log

**Comment by pavlos1982** [comment]
I am using version 2.3.70 of security onion, I re-downloaded the latest version (2.3.70-WAZUH ISO image) all installed successfully, but this time around the same  problem when I monitor via ungroup it works for a while then the alert stop showing but however when using alert by (Group By Sensor, Source IP/Port, Destination IP/Port, Name) I do see it is showing the alert but my issue with using this alert is there is no time with it that is why I want to use ungroup that gives me the time of the alert see the screenshots attached.

the installation of security onion I used on a stand alone using a pendrive that copied security onion with balenaEtcher software as recommended on the security onion website.

installation type I used evaluation mode for testing first but then I wanted to use a standalone for my environment.

all services are running without any errors see screenshot

when running salt-call state.highstate no errors see screenshot:

![saltstate](https://user-images.githubusercontent.com/5363713/131967814-1fbae2e2-0e4e-45ea-938f-bd7678134473.JPG)


no failures on the grid status



![sec2](https://user-images.githubusercontent.com/5363713/131966119-1f84a807-296d-4237-b82a-313261b5b2ef.JPG)


![sec1](https://user-images.githubusercontent.com/5363713/131966110-78ab2661-3b78-4b51-b5ac-234458880aaa.JPG)

![secstatus](https://user-images.githubusercontent.com/5363713/131967145-fabbe2ad-42da-45cb-b70a-26c29c9cf01b.JPG)

**Comment by pavlos1982** [comment]
I want to be able to do live monitoring that includes all of the alerts, it seem to work for few hours then stop

**Comment by dougburks** [comment]
> when I monitor via ungroup it works for a while then the alert stop showing but however when using alert by (Group By Sensor, Source IP/Port, Destination IP/Port, Name) I do see it is showing the alert but my issue with using this alert is there is no time with it that is why I want to use ungroup that gives me the time of the alert see the screenshots attached.

I'm not sure I understand your description of the problem. Are you asking about the `Listened ports status` alert that is highlighted in the ungrouped screenshot? This is a Wazuh HIDS alert generated by the Security Onion system (as opposed to a NIDS alert generated by network traffic). Because of this, there is no IP address or port information and therefore it won't be shown when doing a Group By that requires that information.

For more information, please see:
https://docs.securityonion.net/en/2.3/wazuh.html
https://docs.securityonion.net/en/2.3/alerts.html

**Comment by pavlos1982** [comment]
before I receiving these alerts when I select the ungroup option under alerts:

![sec12](https://user-images.githubusercontent.com/5363713/131994528-d54f347c-d2aa-4a66-ada5-9db0b6f1e403.JPG)

and now I only see this

![sec11](https://user-images.githubusercontent.com/5363713/131994568-64664002-c7fa-4b7f-8a30-8c579669c0e3.JPG)

is there a way I can see all the HIDS alerts including the IP  source and destination, plus the time and date? I just dont understand why before it was showing live alerts with time relevant to my current PC+ source ip and destination ip , then suddenly it stops

regards,

Paul

**Comment by pavlos1982** [comment]
After messing around with the settings I noticed this, when set to 8 hours the alerts shows but if I set it back to 24 hours no alerts, see the screenshots. Is there some-kind of limit on the logs? or is it maybe that it is using the CentOS time instead of the browser time ?

![alert1](https://user-images.githubusercontent.com/5363713/132067485-1d9d71fd-9751-47e8-b627-c3068c5dad8b.JPG)
![alert2](https://user-images.githubusercontent.com/5363713/132067492-a6668e19-3035-4ad3-bf98-8029c1f982f5.JPG)

**Comment by dougburks** [comment, marked as answer]
In the Ungrouped view, scroll all the way down to the bottom of the page and you should see a `Rows per page` selection. When you adjust this value, it should adjust the `el` parameter in the URL, so you can also change this manually in the URL.

**Comment by pavlos1982** [comment]
Hi Doug,

Thanks for getting back to me, unfortunately in the  URL I am not seeing any el see screenshot, again if you noticed between the two screenshot when I use 8 hours I am able to capture alerts for 8 hours to the current time, but when I extend this to 24 hours or above 8 hours it gives me a different result and I can only capture some alerts and it is also behind the my current time, thanks

![al11](https://user-images.githubusercontent.com/5363713/132400602-7889544c-ea1f-4dbf-9c0a-5d1e943f4cf9.JPG)
![al22](https://user-images.githubusercontent.com/5363713/132400606-350ec10c-f477-4335-9b90-fef77df90fb2.JPG)

**Reply by dougburks** [reply]
Looking at your screenshot, I see `el=1000` near the end of blue highlighted section of the URL.

**Comment by pavlos1982** [comment]
Hi Doug,

Thanks got it lol, I must be going blind, increasing the el now I see the most recent alert, thank you, something else I noticed is there a way to prevent security onion to time out to keep the alert screen active or is this base on the browser itself? it seems after a while it goes inactive not sure if this is to do with the ssl certificate or something else?

**Reply by dougburks** [reply]
It seems we have answered this discussion about alerts. If you have other questions or problems, please start a separate discussion with appropriate title and provide detailed information there.

**Reply by pavlos1982** [reply]
Thank you for all your help Dough, I will start a new discussion.
